### DISCUSSION TITLE
Add favourite settings for multiple different training methods

### DISCUSSION THREAD

**Discussion by RayMacLewis** [monkeytypegame/monkeytype#1450] [Ideas] [answered]
Hey guys and gals,

I would love to be able to save my settings for multiple different training methods.
This way one you integrate shortcut-links specific for each user.

For example:
- one day I want to train plain english
- the other day I want to train coding in python

Let me know what you think about that!
Ray

**Comment by Miodec** [comment, marked as answer]
Presets should do what you are looking for. Look for them in the settings.

**Reply by RayMacLewis** [reply]
Unbelievable! You really thought about every little detail I would ever wanted to integrate in a typing website. Fantastic work!

edit: Can be closed of course!

**Reply by unrealapex** [reply]
Monkeytype is the typing website of your dreams. Everything you want, nothing you don't. @Miodec and [all other contributors](https://github.com/Miodec/monkeytype/graphs/contributors) have made this the de facto standard for typing tests for users all around the globe.
